### PR TITLE
Add mongodb language support

### PR DESCRIPTION
**Pull request by Calyhre**
Add mongoDB scripts support:

Atom mongoDB screenshot

### CODE REVIEW

File: .mailmap [hunk 1]
```diff
@@ -2,7 +2,7 @@ Alexey Slaykovsky <alexey@macbook.dlink>  Alexey Slaykovsky <alexey@macbook.dlin
 Andy Hayden <andyhayden1@gmail.com>  Andy Hayden <andyhayden1@gmail.com> 
 Ash Wilson <smashwilson@gmail.com>  Ash Wilson <smashwilson@gmail.com> 
 Calvin Bottoms <calvin.bottoms@gmail.com>  Calvin Bottoms <calvin.bottoms@gmail.com> 
-Calyhre <charley.david@wopata.com>  Calyhre <charley.david@wopata.com> 
+Calyhre <calyhre@gmail.com>  Calyhre <charley.david@wopata.com> 
 Christian Kjaer Laustsen <christianlaustsen@gmail.com>  Christian Kjaer Laustsen <christianlaustsen@gmail.com> 
 Ciaran Downey <me@ciarand.me>  Ciaran Downey <me@ciarand.me> 
 Dan <daniel.bayley@me.com>  Dan <daniel.bayley@me.com>
```
Comment (rgbkrk) [inline .mailmap]: Thanks for updating the mailmap! I don't always get to this.

File: package.json [hunk 2]
```diff
@@ -64,7 +64,7 @@
     },
     {
       "name": "Calyhre",
-      "email": "charley.david@wopata.com"
+      "email": "calyhre@gmail.com"
```
Comment (rgbkrk) [inline package.json]: The contributors actually get auto-updated with a script before release, but this is fine.

**Comment by rgbkrk** [comment]
:shipit:

```
$ apm publish patch
Preparing and tagging a new version ✓
Pushing v2.25.2 tag ✓
Publishing script@v2.25.2 ✓
```

**Comment by Calyhre** [comment]
:raised_hands:  Thanks :)

**Pull request merged**